k8_cortx_cloud/solution.yaml

```diff
@@ -3,12 +3,12 @@ solution:
   secrets:
     name: cortx-secret
     content:
-      kafka_admin_secret: null
-      consul_admin_secret: null
-      common_admin_secret: null
-      s3_auth_admin_secret: null
-      csm_auth_admin_secret: null
-      csm_mgmt_admin_secret: null
+      kafka_admin_secret: Seagate@123
+      consul_admin_secret: Seagate@123
+      common_admin_secret: Seagate@123
+      s3_auth_admin_secret: cortxadmin
```

Comment:
In 0.1.0 this is `ldapadmin`. Any concern that someone may also be hard-coding that one, and switching to `cortxadmin` would be a problem?